### PR TITLE
Add "My projects" menu item to user menu.

### PR DESCRIPTION
**Pull request by larrypo**
This works for both dashboard and pegasus. It goes to the same page as the "Projects" button,
as well as the "View all projects..." link at the bottom of the Create menu.

Also, this does not introduce the "My projects" string to the pegasus spreadsheet. If this seems to be the correct approach,
I can make the spreadsheet change first.

Screen Shot 2019-04-05 at 2 28 26 PM

### CODE REVIEW

**Comment by dju90** [comment]
What spreadsheet does "Also, this does not introduce the pegasus spreadsheet. If this seems to be the correct approach, I can make the spreadsheet change first" refer to?

File: pegasus/cache/i18n/en-US.yml
Comment (breville) [inline pegasus/cache/i18n/en-US.yml]: Yes, this will be overwritten when we do a content scoop on the staging machine as we pull changes made to the gsheet.  Instead, this should be modified in the gsheet  first,and we should ensure that the staging branch contains the scooped change before this PR is merged.

**Review by Hamms — APPROVED** [review]
LGTM, pending confirmation that the pegasus i18n string has been added to the gsheet

File: pegasus/cache/i18n/en-US.yml
Comment (larrypo) [reply pegasus/cache/i18n/en-US.yml]: OK, I've added it to the spreadsheet, line 1492 (http://sheets/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU#gid=0)

Comment (breville) [reply pegasus/cache/i18n/en-US.yml]: Great.  You can confirm everything has worked by waiting for that change to be scooped, and then merge the staging branch into this branch, and the above line should vanish.  (If it's still there, we'll have to remove it so we don't have a double-entry.)

Comment (larrypo) [reply pegasus/cache/i18n/en-US.yml]: I've merged with head, and it looks to me like this change has gone away, so we should be good to go.

**Review by breville — APPROVED** [review]

**Comment by breville** [comment]
Drone tests passed.  I'll merge now.

**Pull request merged**
